src/app/admin/invite/invite-form.tsx

```diff
@@ -36,7 +36,12 @@ export function InviteForm() {
         },
         onSuccess(info) {
             notifications.show({ message: `User invited successfully\nClerk ID: ${info.clerkId}`, color: 'green' })
-            studyProposalForm.setValues(initialValues())
+            // Keep the selected organization, reset other fields using initialValues
+            const currentOrgId = studyProposalForm.values.organizationId
+            const newInitialValues = initialValues()
+            newInitialValues.organizationId = currentOrgId // Restore the organization
+            studyProposalForm.setValues(newInitialValues)
+            studyProposalForm.clearErrors() // Clear validation state explicitly
```

Comment:
https://mantine.dev/form/use-form/

there is a method to do this simpler i think:
```
// Resets form values to `initialValues`,
// clears all validation errors,
// resets touched and dirty state
form.reset();

```

Reply:
Also initialValues can just be moved inline, rather than being a function up above ^

Reply:
I think I've got it working with reset now. There was some weird behaviour (probably because I wanted to keep the organisation) but reset() works now in my testing.